alws/crud/build_node.py

```diff
@@ -162,6 +213,29 @@ async def __process_logs(pulp_client: PulpClient, task_id: int,
                     href=href
                 )
             )
+    if exceptions:
+        raise ArtifactConversionError(
+            'Cannot put build logs into Pulp storage: %s',
+            '\n'.join([str(e) for e in exceptions]))
+
+    for repo_href, files in repo_files_mapping.items():
+        try:
+            await pulp_client.modify_repository(repo_href, add=files)
```

Comment:
Same as above, can be made in parallel, publication can be removed

Reply:
Yeah, found the thing about publication, removing that call